test/Microsoft.ServiceHub.Analyzers.Tests/Microsoft.ServiceHub.Analyzers.Tests.csproj

```diff
@@ -14,6 +14,12 @@
     <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.CodeFix.Testing.XUnit" />
     <PackageReference Include="Microsoft.CodeAnalysis" />
     <PackageReference Include="NuGet.Protocol" />
+
+    <!--
+      This is a transitive dependency of Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.XUnit and Microsoft.CodeAnalysis.VisualBasic.CodeFix.Testing.XUnit.
+      Those packages reference verison 6.0.0 of this pacakge which has a vulnerability
+    -->
+    <PackageReference Include="System.Formats.Asn1" />
```

Comment:
We don't need these references, because we have transitive pinning turned on. All that is required is the `PackageVersion` item.